### ISSUE TITLE
update minimum example in README?

### ISSUE THREAD

**Issue by hainm** [ParmEd/ParmEd#717]
since you suggested users to use new syntax several times, what's about give an example in READMe file?

```python
import parmed as pmd

gmx_top = pmd.load_file('pmaawaterFE20mer2.top',
                         xyz='pmaawaterFE20mer2.gro')
gmx_top.save('pmaa.top', format='amber')
gmx_top.save('pmaa.crd', format='rst7')
```

**Comment by swails** [comment]
Good idea. I'm open to PRs as my time right now is quite limited.

**Comment by hainm** [comment]
Ok. I can do that.

**Comment by swails** [comment]
Fixed in #741
